tests/util/kube_utils.go

```diff
@@ -792,14 +792,8 @@ func CreateMultiClusterSecrets(namespace string, KubeClient kubernetes.Interface
 				ServerEndpoints: nil,
 				CABundle:        nil,
 			},
-			AuthInfo: v1alpha1.AuthInfo{
-				Providers: nil,
-			},
-			CloudProvider: &v1alpha1.CloudProvider{
-				Name: "",
-			},
+			AuthInfo: v1alpha1.AuthInfo{},
```

Comment:
What is this for ? If 1.9 needs what you removed put it back. We need to support the last 3 release of k8s.

Reply:
The whole reason for this PR is that K8s 1.10 _vendor/k8s.io/client-go/rest/config.go:81_ adds `ExecProvider *clientcmdapi.ExecConfig`.  Many configs need that the client-go code to use that provider.  1.10 removed these alpha version fields and the client code does not compile with them.

We should be able to talk to any K8s version that _kubectl_ can talk to with this change.